lib/queue.rb

```diff
@@ -1,28 +1,61 @@
+# Wave 2
+# require_relative './stack.rb'
+require "./linked_list"
+
 class Queue
 
   def initialize
-    # @store = ...
-    raise NotImplementedError, "Not yet implemented"
+    @store = LinkedList.new
```

Comment:
⚠️ You were asked to use a circular buffer instead of a LinkedList for this class. 

These work, but using a LinkedList wasn't the assignment.